examples/canvas_lines_dashed.html

```diff
@@ -72,13 +72,14 @@
 
 				var spline = new THREE.CatmullRomCurve3( points );
 				var geometrySpline = new THREE.Geometry();
+				var point = new THREE.Vector3();
 
 				for ( var i = 0; i < points.length * subdivisions; i ++ ) {
 
 					var index = i / ( points.length * subdivisions );
-					var position = spline.getPoint( index );
+					spline.getPoint( index, point );
 
-					geometrySpline.vertices[ i ] = new THREE.Vector3( position.x, position.y, position.z );
+					geometrySpline.vertices[ i ] = new THREE.Vector3( point.x, point.y, point.z );
```

Comment:
Why not this, instead?
```javascript
var t = i / ( points.length * subdivisions );
geometrySpline.vertices[ i ] = spline.getPoint( t );
```

Reply:
Indeed, much better 😊

Reply:
I believe there are multiple occurrences of this you can fix.

Reply:
I think it should be okay now...